docs/modules/user-guide/partials/proc_adding-a-name-to-a-devfile.adoc

```diff
@@ -2,7 +2,13 @@
 = Adding a name to a devfile
 
 [role="_abstract"]
-Adding a name to a devfile is mandatory. Use the `name` or `generateName` attributes (or both) to define the name.
+Adding a name to a devfile is mandatory. Use the `name` or `generateName` attributes to define the name. See the following list for metadata properties in a devfile:
```

Comment:
Remove `generateName` from this page (including the example) since we don't have generateName support anymore.